scss/_settings_colors.scss

```diff
@@ -55,23 +60,34 @@ $active-background-opacity-percentage: $active-background-opacity-amount * 100%;
 $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
 
 // NON-SEMANTIC COLOURS
-$color-label-validated: #006b75;
-$color-code-background: rgba($color-x-dark, 0.03);
-$color-code-background-dark: rgba($color-x-light, 0.3);
-$color-code-heading-background: rgba($color-x-dark, 0.08);
+// TODO: This is unused
+$color-label-validated: $vf-color-non-semantic-label-validated;
```

Comment:
I'm not sure at what point and why these were called "non-semantic", this may be quite a legacy header.

I'd rather not upstream it to tokens naming. Make it part of "base" or whatever. Creating a separate group doesn't seem to have much value I think.